### PR TITLE
doc: Minor language improvement in TLS documentation

### PR DESCRIPTION
**Pull request by sharmarajdaksh**
### What does this PR do?

- Makes minor language improvements in the TLS documentation page

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/https/tls.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ spec:
 
 ### Strict SNI Checking
 
-With strict SNI checking, Traefik won't allow connections from clients connections
+With strict SNI checking enabled, Traefik won't allow connections from clients connections
```
Comment (SantoDE) [inline docs/content/https/tls.md:319]: suggest removing the last connections in this line

**Review by SantoDE — CHANGES_REQUESTED** [review]

File: docs/content/https/tls.md (hunk 1 above)
Comment (sharmarajdaksh) [reply docs/content/https/tls.md]: Agree with your suggestion, made the change

**Review by rtribotte — APPROVED** [review]
LGTM 👍

**Review by SantoDE — APPROVED** [review]
LGTM 👼

**Review by ldez — APPROVED** [review]
Thanks :+1:

**Pull request merged**